src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -2502,9 +2504,10 @@ private bool ConsumeMultiLineCommentMultiSegment(ReadOnlySpan<byte> localBuffer,
         Done:
             if (HasValueSequence)
             {
-                // Include the final slash at the end of the comment as part of it.
-                SequencePosition end = new SequencePosition(_currentPosition.GetObject(), _currentPosition.GetInteger() + i + 1);
+                SequencePosition end = new SequencePosition(_currentPosition.GetObject(), _currentPosition.GetInteger() + i);
                 ReadOnlySequence<byte> commentSequence = _sequence.Slice(start, end);
+                // Exclude the /* and the trailing *
+                commentSequence = commentSequence.Slice(2, commentSequence.Length - 3);
```

Comment:
Same here. Can we set start/end such that this slice isn't required?

Reply:
Working on this.